pkg/systemd/superuser-alert.jsx

```diff
@@ -50,7 +52,8 @@ export class SuperuserAlert extends React.Component {
             <>
                 <Alert className="ct-limited-access-alert"
                        hidden={this.superuser.Current != "none"}
-                       variant="info" isInline
+                       variant="warning" isInline
```

Comment:
I did not take a look in the CSS changes yet, but I am not convinced that it's correct to show a 'warning' alert for not privileged logins..

Reply:
I'm a bit torn here. On one hand being not root is the expected mode for SSH/desktop sessions, and some users are not even allowed to become root (there is no way to predict that). OTOH being root is the main use case for cockpit..

But even if we dial this back to info, I find all the other changes really excellent! Thanks @garrett!

Reply:
It doesn't have a warning icon, but a custom one, however. 

The blue alert with another blue alert (login, which should _not_ be an alert, but currently is) is an excellent way to completely ignore the alert, which is part of the problem.

The "warning" is that Cockpit will not work the way you expect, if you generally use Cockpit as an admin, which most people do. It can be ignored by going to other pages.

Right now, Cockpit is quite broken in several pages when you're not the admin. (Pages don't let you know and may be blank or partially blank without any fallback message.) So it is indeed a warning and important to know to switch (if you can). We should also fix these pages, but it's still important regardless, as it will forever be important to be an admin to perform admin tasks.

Reply:
I'd like to point out that we had people in our UX study from a while back who missed not just the top shell area, but also the notification. It wasn't eye-catching enough. Part of that is the understated blue color (including its repetition with the log in "alert").

Reply:
Thanks for the detailed answer. Since we only borrowed the 'warning' color and not the icon as well, I think indeed we are walking on the middle ground of raising attention but not really marking it as a warning. I like it, thanks for this improvement.